deploy/credential_request.yaml

```diff
@@ -22,4 +22,6 @@ spec:
       - s3:PutBucketTagging
       - s3:DeleteObjectTagging
       - s3:GetBucketTagging
+      - tag:GetResources
```

Comment:
Is this scope needed? I don't see a use of the direct tag client?

Reply:
I'll try without it and see what happens. It could have been left over from using the resource tagging API.

Reply:
Looks like this isn't needed. Good catch!